pkg/virt-controller/watch/workload-updater/workload-updater_test.go

```diff
@@ -427,8 +396,8 @@ var _ = Describe("Workload Updater", func() {
 			controller.Execute()
 			Expect(recorder.Events).To(BeEmpty())
 
-			// sleep to account for batch interval
-			time.Sleep(3 * time.Second)
+			// Shift time for batch interval
+			controller.lastDeletionBatch = time.Now().Add(-3 * time.Second)
```

Comment:
Great catch :) Getting rid of `time.Sleep` looks like an awesome improvement.

Reply:
Funny enough it was only slight impact.